src/Generator.php

```diff
@@ -325,6 +325,8 @@ private function addActionDefinitions()
 
     private function getRouteUri()
     {
-        return Str::replaceFirst($this->config['basePath'], '', $this->route->uri());
+        $uri = Str::replaceFirst($this->config['basePath'], '', $this->route->uri());
+
+        return Str::start($uri, '/');
```

Comment:
fixed bug where if the user added `/` as basePath it would strip out the leading `/` from the path in the swagger docs